src/main/resources/clojure/rules.xml

```diff
@@ -404,4 +404,101 @@
         <remediationFunction>CONSTANT_ISSUE</remediationFunction>
         <remediationFunctionBaseEffort>60min</remediationFunctionBaseEffort>◊
     </rule>
+    <rule>
+        <key>unused-binding</key>
+        <name>unused-binding</name>
+        <internalKey>unused-binding</internalKey>
+        <description>clj-kondo linting rule
```

Comment:
Does kondo provide descriptions for this rules? Maybe we could just reuse them. I didn't have time to that for eastwood, maybe we could do it right for Kondo. hehe

Reply:
done